spec/controllers/donations_controller_spec.rb

```diff
@@ -22,6 +22,15 @@
       it "returns http success" do
         expect(subject).to be_successful
       end
+
+      context "when creating a new donation" do
```

Comment:
Can this be moved to requests instead of controllers? Those are kind of outmoded.